package.json

```diff
@@ -16,7 +16,9 @@
   },
   "scripts": {
     "algolia:download": "algolia getsettings -a $ALGOLIA_APP_ID -k $ALGOLIA_API_KEY -n $ALGOLIA_RESEARCH_OUTPUT_INDEX | prettier --parser json > ./ci/algolia-schema.json",
+    "prealgolia:upload": "sh ./ci/config-algolia-replicas.sh",
     "algolia:upload": "algolia setsettings -a $ALGOLIA_APP_ID -k $ALGOLIA_API_KEY -n $ALGOLIA_RESEARCH_OUTPUT_INDEX -s ./ci/algolia-schema.json",
```

Comment:
Can we change this so we only need to run algolia:upload? All other commands should be started by this command if we are going to need a few for other rankings

Reply:
I think we need the three commands. But happy to hear suggestions:) 
Before running `algolia:upload` we need to inject the value of the var `$ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA` into `algolia-schema.json`.  
And then, we need to load the config of the replica.